### ISSUE TITLE
Results on Cifar100 dataset

### ISSUE THREAD

**Issue by aynesss** [zhangzhao156/Communication-efficient-federated-continual-learning#3]
Hello, I run main_fesdsi with cifar 100, and i split the dataset to 10 tasks among 50 clients, I use hyperparameters E=10, batch_size=2, lambda=1.0, epsilon=0.0001, 
In the first round , I found:

` interation 0 client 0
Train Epoch:1	Loss:4.3056	EWC_Loss:0.0000	CE_Loss:4.3056	Accuracy: 8.2000
Train Epoch:2	Loss:3.9568	EWC_Loss:0.0000	CE_Loss:3.9568	Accuracy: 22.5000
Train Epoch:3	Loss:1.8762	EWC_Loss:0.0000	CE_Loss:1.8762	Accuracy: 27.3000
Train Epoch:4	Loss:2.4257	EWC_Loss:0.0000	CE_Loss:2.4257	Accuracy: 30.2000
Train Epoch:5	Loss:2.3992	EWC_Loss:0.0000	CE_Loss:2.3992	Accuracy: 28.5000
Train Epoch:6	Loss:4.5117	EWC_Loss:0.0000	CE_Loss:4.5117	Accuracy: 31.0000
Train Epoch:7	Loss:2.0042	EWC_Loss:0.0000	CE_Loss:2.0042	Accuracy: 32.6000
Train Epoch:8	Loss:2.0558	EWC_Loss:0.0000	CE_Loss:2.0558	Accuracy: 31.2000
Train Epoch:9	Loss:0.8134	EWC_Loss:0.0000	CE_Loss:0.8134	Accuracy: 34.6000
Train Epoch:10	Loss:1.5176	EWC_Loss:0.0000	CE_Loss:1.5176	Accuracy: 32.9000
interation 0 client 1
Train Epoch:1	Loss:3.9777	EWC_Loss:0.0000	CE_Loss:3.9777	Accuracy: 9.9000
Train Epoch:2	Loss:4.2920	EWC_Loss:0.0000	CE_Loss:4.2920	Accuracy: 22.0000
Train Epoch:3	Loss:4.9061	EWC_Loss:0.0000	CE_Loss:4.9061	Accuracy: 23.1000
Train Epoch:4	Loss:2.2826	EWC_Loss:0.0000	CE_Loss:2.2826	Accuracy: 27.6000
Train Epoch:5	Loss:1.8088	EWC_Loss:0.0000	CE_Loss:1.8088	Accuracy: 27.8000
Train Epoch:6	Loss:4.5937	EWC_Loss:0.0000	CE_Loss:4.5937	Accuracy: 31.6000
Train Epoch:7	Loss:1.7471	EWC_Loss:0.0000	CE_Loss:1.7471	Accuracy: 31.1000
Train Epoch:8	Loss:1.7462	EWC_Loss:0.0000	CE_Loss:1.7462	Accuracy: 30.4000
Train Epoch:9	Loss:1.6675	EWC_Loss:0.0000	CE_Loss:1.6675	Accuracy: 30.3000
Train Epoch:10	Loss:3.0170	EWC_Loss:0.0000	CE_Loss:3.0170	Accuracy: 32.9000
interation 0 client 2
Train Epoch:1	Loss:4.5627	EWC_Loss:0.0000	CE_Loss:4.5627	Accuracy: 8.2000
Train Epoch:2	Loss:4.0515	EWC_Loss:0.0000	CE_Loss:4.0515	Accuracy: 23.6000
Train Epoch:3	Loss:4.2134	EWC_Loss:0.0000	CE_Loss:4.2134	Accuracy: 25.6000
Train Epoch:4	Loss:2.0662	EWC_Loss:0.0000	CE_Loss:2.0662	Accuracy: 29.3000
Train Epoch:5	Loss:3.8550	EWC_Loss:0.0000	CE_Loss:3.8550	Accuracy: 30.1000
Train Epoch:6	Loss:5.4653	EWC_Loss:0.0000	CE_Loss:5.4653	Accuracy: 27.5000
Train Epoch:7	Loss:4.7028	EWC_Loss:0.0000	CE_Loss:4.7028	Accuracy: 30.4000
Train Epoch:8	Loss:4.6260	EWC_Loss:0.0000	CE_Loss:4.6260	Accuracy: 30.5000
Train Epoch:9	Loss:2.3339	EWC_Loss:0.0000	CE_Loss:2.3339	Accuracy: 32.7000
Train Epoch:10	Loss:1.3589	EWC_Loss:0.0000	CE_Loss:1.3589	Accuracy: 32.8000
interation 0 client 3
Train Epoch:1	Loss:4.6276	EWC_Loss:0.0000	CE_Loss:4.6276	Accuracy: 6.8000
Train Epoch:2	Loss:3.4649	EWC_Loss:0.0000	CE_Loss:3.4649	Accuracy: 24.3000
Train Epoch:3	Loss:1.1430	EWC_Loss:0.0000	CE_Loss:1.1430	Accuracy: 26.8000
Train Epoch:4	Loss:2.1834	EWC_Loss:0.0000	CE_Loss:2.1834	Accuracy: 27.7000
Train Epoch:5	Loss:3.8328	EWC_Loss:0.0000	CE_Loss:3.8328	Accuracy: 27.3000
Train Epoch:6	Loss:4.7345	EWC_Loss:0.0000	CE_Loss:4.7345	Accuracy: 31.3000
Train Epoch:7	Loss:4.5733	EWC_Loss:0.0000	CE_Loss:4.5733	Accuracy: 32.5000
Train Epoch:8	Loss:4.5085	EWC_Loss:0.0000	CE_Loss:4.5085	Accuracy: 30.9000
Train Epoch:9	Loss:2.5440	EWC_Loss:0.0000	CE_Loss:2.5440	Accuracy: 28.8000
Train Epoch:10	Loss:2.4723	EWC_Loss:0.0000	CE_Loss:2.4723	Accuracy: 32.9000
interation 0 client 4
Train Epoch:1	Loss:4.0241	EWC_Loss:0.0000	CE_Loss:4.0241	Accuracy: 10.4000
Train Epoch:2	Loss:5.4023	EWC_Loss:0.0000	CE_Loss:5.4023	Accuracy: 23.1000
Train Epoch:3	Loss:3.0204	EWC_Loss:0.0000	CE_Loss:3.0204	Accuracy: 24.9000
Train Epoch:4	Loss:2.7128	EWC_Loss:0.0000	CE_Loss:2.7128	Accuracy: 27.0000
Train Epoch:5	Loss:1.5448	EWC_Loss:0.0000	CE_Loss:1.5448	Accuracy: 28.6000
Train Epoch:6	Loss:2.3282	EWC_Loss:0.0000	CE_Loss:2.3282	Accuracy: 29.5000
Train Epoch:7	Loss:6.0247	EWC_Loss:0.0000	CE_Loss:6.0247	Accuracy: 32.6000
Train Epoch:8	Loss:5.7092	EWC_Loss:0.0000	CE_Loss:5.7092	Accuracy: 31.4000
Train Epoch:9	Loss:3.2523	EWC_Loss:0.0000	CE_Loss:3.2523	Accuracy: 31.6000
Train Epoch:10	Loss:3.5057	EWC_Loss:0.0000	CE_Loss:3.5057	Accuracy: 32.1000

all_precision 0.224517868280087 all_recall 0.292 all_fscore 0.18879574710613442
[[100   0   0   0   0   0   0   0   0   0]
 [ 52   0   1  32   0   0   0  14   1   0]
 [ 14   0  75  10   0   0   0   1   0   0]
 [  7   0   0  89   0   0   0   4   0   0]
 [  8   0   1  88   0   0   0   3   0   0]
 [ 28   0   1  66   0   0   0   5   0   0]
 [  2   0   0  93   0   0   0   5   0   0]
 [ 51   0   0  22   0   0   0  27   0   0]
 [ 72   0   0  12   0   0   0  15   1   0]
 [ 90   0   0   3   0   0   0   7   0   0]]

Test set: Average loss: 1.3157 
Accuracy: 292/1000 29.20
Testing accuracy: 29.20 `

Can you explain this result please
